src/main/java/com/klin/holoItems/collections/gamers/okayuCollection/items/MoguMogu.java

```diff
@@ -0,0 +1,187 @@
+package com.klin.holoItems.collections.gamers.okayuCollection.items;
+
+import com.klin.holoItems.HoloItems;
+import com.klin.holoItems.Item;
+import com.klin.holoItems.abstractClasses.Enchant;
+import com.klin.holoItems.collections.gen1.melCollection.items.ReadingGlasses;
+import com.klin.holoItems.collections.gen4.cocoCollection.items.DragonHorns;
+import com.klin.holoItems.collections.gen5.botanCollection.items.Backdash;
+import com.klin.holoItems.interfaces.Consumable;
+import com.klin.holoItems.interfaces.Hungerable;
+import com.klin.holoItems.utility.Utility;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.Statistic;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.entity.FoodLevelChangeEvent;
+import org.bukkit.event.player.PlayerItemConsumeEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.PlayerInventory;
+import org.bukkit.inventory.RecipeChoice;
+import org.bukkit.inventory.ShapedRecipe;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+import java.util.*;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class MoguMogu extends Enchant implements Hungerable {
+    public static final String name = "moguMogu";
+    private static final Material material = Material.BEEF;
+    private static final String lore = "Auto-eat food.\n" +
+            "You still need food on your hotbar, though.";
+    private static final int durability = 0;
+    private static final boolean shiny = true;
+    public static final boolean stackable = true;
+    public static final int cost = 64;
+
+    // Accepted armor types this can be on.
+    // Yes, both of these variables represent the same thing; it was written this way in DemonAura.
+    public static final Set<String> acceptedIds = Stream.of(ReadingGlasses.name, DragonHorns.name, Backdash.name).collect(Collectors.toCollection(HashSet::new));
```

Comment:
Should be `Set::of`. This is allocating a stream for no reason. I know you want to say you did it this way because it's how Klin did it, but his coding style ignores modern java language features and makes the code harder to understand/maintain.